dropwizard/build.gradle

```diff
@@ -0,0 +1,20 @@
+plugins {
+    id "com.github.hierynomus.license" version "0.15.0"
+}
+
+//license {
+//  header project.file('../CODE-LICENSE.txt')
+//  strictCheck true
+//   exclude "**/*.properties"
+//}
+
+
+dependencies {
+    compile project(':jetty')
```

Comment:
Dropwizard includes its own Jetty. Need to look into having exclusions.

Reply:
That's interesting. Does it ship its own Jetty fork, or does it depend on a specific Jetty version? Exclusion can be done like this:

```gradle
dependencies {
    compile(project(':jetty')) {
        exclude group: 'org.eclipse.jetty', module: 'jetty-server'
        exclude group: 'org.eclipse.jetty', module: '...'
    }
}
```

See https://stackoverflow.com/a/33931450/55808 for more information.

Reply:
It just has a transitive dependency on Jetty and other libraries - http://dropwizard.io/en/stable/getting-started.html

I know how to exclude, and I would rather exclude from Dropwizard if Armeria is going to take over that interaction. I meant I need to look if the versions are indeed different, or Gradle is smart enough to take care of that.

Reply:
I see. I'd prefer using the Jetty version we provide, unless there's a conflict. Armeria usually depends on the latest version of Jetty, so I guess Gradle will choose what `armeria-jetty` depends on. To check the resolved dependency versions, you can use `./gradlew :dropwizard:dependencies`. (Let me know if you already know about this, so I don't provide you too much information. :wink:)

Reply:
Always looking to learn more Gradle after dealing with Maven during the day 🤣

Reply:
Looks like it pulls in the upgraded Jetty version

```
|    |    +--- io.dropwizard:dropwizard-lifecycle:1.3.16
|    |    |    +--- org.slf4j:slf4j-api:1.7.26 -> 1.7.28
|    |    |    +--- com.google.guava:guava:24.1.1-jre -> 28.1-jre (*)
|    |    |    +--- org.eclipse.jetty:jetty-server:9.4.18.v20190429 -> 9.4.22.v20191022 (*)
|    |    |    \--- io.dropwizard:dropwizard-util:1.3.16 (*)
```